src/reason-parser/reason_pprint_ast.ml

```diff
@@ -6888,8 +6889,19 @@ class printer  ()= object(self:'self)
     | `NormalFunAppl args ->
         let theFunc = SourceMap (funExpr.pexp_loc, self#simplifyUnparseExpr funExpr) in
         (*reset here only because [function,match,try,sequence] are lower priority*)
+        let syntheticArgLoc = match args with
```

Comment:
Here's where I synthesize a location for the arguments list `call(a, b)`. There is not actually a tuple, so there wasn't a location - so I constructed one from the information available.